ChangeLog

```diff
@@ -34,6 +34,9 @@ Bugfix
      with flag MBEDTLS_X509_BADCERT_BAD_PK even when the key type was correct.
      In the context of SSL, this resulted in handshake failure. #1351
    * Fix Windows x64 builds with the included mbedTLS.sln file. #1347
+   * In test_suite_pk pass valid parameters when testing for hash length 
+     overflow. #1179
+   * Fix Windows x64 builds with the included mbedTLS.sln file. #1347
```

Comment:
Why does your commit add `Fix Windows x64 builds with the included mbedTLS.sln file. #1347`? I don't think your commit should add this, as it doesn't fix Windows x64 builds.

Reply:
Sorry this was a mistake, the comment 'Fix Windows x64 builds with the included mbedTLS.sln file. #1347' was duplicated by mistake, fixing this.